### PR TITLE
Update libdeflate to 1.18

### PR DESCRIPTION
**Pull request by jakobnissen**

### CODE REVIEW

File: L/libdeflate/build_tarballs.jl [hunk 1]
```diff
@@ -3,20 +3,22 @@
...
+    ),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libdeflate-*/
-make PROG_SUFFIX=$exeext PREFIX=${prefix} LIBDIR=${libdir} DISABLE_ZLIB=true
-make PROG_SUFFIX=$exeext PREFIX=${prefix} LIBDIR=${libdir} install
+cd $WORKSPACE/srcdir/libdeflate
+cmake -B build && cmake --build build
```
Comment (giordano) [inline L/libdeflate/build_tarballs.jl:20]: This is wrong: https://docs.binarybuilder.org/stable/build_tips/#CMake-builds

File: L/libdeflate/build_tarballs.jl [hunk 2]
```diff
@@ -3,20 +3,36 @@
...
+        "0f03869f72df8705b832910517b47dd5b79eb4e160512602f593ed243b28715f"
+    ),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libdeflate-*/
-make PROG_SUFFIX=$exeext PREFIX=${prefix} LIBDIR=${libdir} DISABLE_ZLIB=true
-make PROG_SUFFIX=$exeext PREFIX=${prefix} LIBDIR=${libdir} install
+# This requires macOS 10.14
```
Comment (giordano) [inline L/libdeflate/build_tarballs.jl:22]: On slack you said you require 10.13, not 10.14

**Review by giordano — COMMENTED** [review]
Now that you have two sources you have to manually install the license file: https://docs.binarybuilder.org/stable/build_tips/#Installing-the-license-file

File: L/libdeflate/build_tarballs.jl (hunk 2 above)
Comment (jakobnissen) [reply L/libdeflate/build_tarballs.jl]: Right - I forgot to change it to 10.13 after it worked! Thank you for the help

**Pull request merged**